golang/Makefile

```diff
@@ -27,6 +27,7 @@ clean:
 	echo Cleaning build artifacts...
 	go clean
 	rm -rf $(BUILD)
+	rm -rf examples/thrift/gen-go
```

Comment:
nit: generally preferable to put generated sources in a different directory from hand-written code.  failing that, please add a .gitignore line to ensure that generated sources aren't accidentally committed to the repo.